configure.ac

```diff
@@ -128,8 +128,8 @@ AC_ARG_ENABLE(openssl_tests,
 
 AC_ARG_ENABLE(experimental,
     AS_HELP_STRING([--enable-experimental],[allow experimental configure options [default=no]]),
-    [use_experimental=$enableval],
-    [use_experimental=no])
+    [enable_experimental=$enableval],
+    [enable_experimental=no])
```

Comment:
Hm I was wondering why this has been working then since ever. The reason is that `enable_experimental` is automatically assigned, and so we don't need to set this at all here...

See  https://www.gnu.org/software/autoconf/manual/autoconf-2.70/html_node/Package-Options.html or https://autotools.io/autoconf/arguments.html .

And there's no reason to have sometimes `use` and sometimes `enable`. we may want to do some cleanup in our `configure.ac`. But that does not need to happen in this PR.